spack_environments/std_versions_and_variants_llnl_lc_cz.yaml

```diff
@@ -43,3 +43,47 @@
     zlib::
       buildable: True
       version: [1.2.11]
+
+  modules::
+    enable::
+      - tcl
+      - lmod
+    lmod::
+      hash_length: 3
+      core_compilers:
+        - 'gcc@4.9.3'
+      naming_scheme: '${PACKAGE}/${VERSION}-${COMPILERNAME}-${COMPILERVER}'
+      hierarchy:
+        - 'mpi'
+        - 'lapack'
+      all:
+        autoload: 'direct'
+        suffixes:
+          '^openblas': openblas
+          '^netlib-lapack': netlib
+          '^python@3.7.2': python-3.7.2
+        filter:
+          # Exclude changes to any of these variables
+          environment_blacklist: ['CPATH', 'LIBRARY_PATH']
+      ^python:
+        autoload:  'direct'
+    tcl:
+      hash_length: 3
+      core_compilers:
+        - 'gcc@4.9.3'
+      naming_scheme: '${PACKAGE}/${VERSION}-${COMPILERNAME}-${COMPILERVER}'
+      whitelist:
+        - gcc
+      blacklist:
+        - '%gcc@4.8'
```

Comment:
We should blacklist GCC less than version 7...

Reply:
Added gcc@4.9.3